src/main/java/com/rocketden/main/game_object/GameTimer.java

```diff
@@ -0,0 +1,45 @@
+package com.rocketden.main.game_object;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.Timer;
+
+/**
+ * The Timer class handles the timing for the Game.
+ * This class is dynamic, so depending on player actions time could
+ * be paused, added, or subtracted.
+ */
+@Getter
+@Setter
+public class GameTimer {
+
+    // 15 minute duration for the GameTimer, in seconds.
+    public static final Long DURATION_15 = (long) 900;
```

Comment:
Is there any specific reason for choosing long here? Not that there's anything wrong with it of course but mainly curious

Reply:
I chose long because the inherent Java functions `plusSeconds` and `schedule` take long as the input, that's all.